src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -1,24 +1,29 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+using System;
+using System.Runtime.InteropServices;
+using System.Text.RegularExpressions;
 using OpenQA.Selenium;
 using OpenQA.Selenium.Chrome;
 using OpenQA.Selenium.Remote;
-using System;
 using Xunit.Abstractions;
+using Xunit.Sdk;
 
-namespace Microsoft.AspNetCore.Components.E2ETest.Infrastructure
+namespace Microsoft.AspNetCore.E2ETesting
 {
     public class BrowserFixture : IDisposable
     {
-        public IWebDriver Browser { get; }
-
-        public ILogs Logs { get; }
+        public BrowserFixture(IMessageSink diagnosticsMessageSink)
+        {
+            DiagnosticsMessageSink = diagnosticsMessageSink;
 
-        public ITestOutputHelper Output { get; set; }
+            if (!HostSupportsBrowserAutomation)
+            {
+                DiagnosticsMessageSink.OnMessage(new DiagnosticMessage("Host does not support browser automation."));
```

Comment:
Is there another test that says how or what one would need to do to resolve this?

Reply:
The build will say it. I'm gonna get rid of this in the next phase